ansible/templates/fail2ban.conf.j2

```diff
@@ -0,0 +1,69 @@
+# Fail2Ban configuration file.
+#
+
+# to view current bans, run one of the following:
+# fail2ban-client status ssh
+# iptables --list -n | fgrep DROP
+
+# The DEFAULT allows a global definition of the options. They can be overridden
+# in each jail afterwards.
+
+[DEFAULT]
+
+ignoreip = 127.0.0.1
+bantime  = 3600
+findtime = 3600
+maxretry = 3
+backend = auto
+usedns = warn
+destemail = {{ ADMIN_EMAIL }}
+
+#
+# ACTIONS
+#
+
+banaction = iptables-multiport
+mta = sendmail
+protocol = tcp
+chain = INPUT
+
+#
+# Action shortcuts. To be used to define action parameter
+
+# The simplest action to take: ban only
+action_ = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
+
+# ban & send an e-mail with whois report to the destemail.
+action_mw = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
+              %(mta)s-whois[name=%(__name__)s, dest="%(destemail)s", protocol="%(protocol)s", chain="%(chain)s"]
+
+# ban & send an e-mail with whois report and relevant log lines
+# to the destemail.
+action_mwl = %(banaction)s[name=%(__name__)s, port="%(port)s", protocol="%(protocol)s", chain="%(chain)s"]
+               %(mta)s-whois-lines[name=%(__name__)s, dest="%(destemail)s", logpath=%(logpath)s, chain="%(chain)s"]
+
+# default action
+action = %(action_mw)s
+
+[ssh]
+
+enabled  = true
+port = 987
+filter   = sshd
+logpath  = /var/log/auth.log
+maxretry = 6
+
+[ssh-ddos]
+
+enabled = true
+port = 987
+filter   = sshd
+logpath  = /var/log/auth.log
+maxretry = 6
+
+[nginx-http-auth]
+
+enabled = true
+filter   = nginx-http-auth
+port     = http,https
+logpath  = /var/log/nginx/error.log
```

Comment:
```suggestion
logpath  = /opt/beryllium/web/nginx/log/error.log
```

Reply:
once this is merged on the master zap-me it should start creating the dir above and the logs as well.

Reply:
just made the change for the logpath to match above... need to test and see if its able to deploy.

Reply:
changed to the suggestion above.